### PR TITLE
moved release_url and created_at to model_build, added support for co…

### PR DESCRIPTION
**Pull request by utkarshgupta95**
moved release_url, created_at to model_build
Added support for component_build_list

### CODE REVIEW

**Comment by aidanheerdegen** [comment]
Related conversations:

https://github.com/ACCESS-NRI/build-cd/pull/63#discussion_r1599307810

https://github.com/ACCESS-NRI/build-cd/pull/63#discussion_r1599344675

File: tools/release_provenance/save_release.py [hunk 1]
```diff
@@ -49,9 +54,10 @@ def get_spack_version(spack_version_data):
 def main():
     release_data_filename = sys.argv[1]
     release_data = read_release_data(release_data_filename)
-    component_build = get_component_build(release_data)
+    model_build = get_model_build(release_data["model_build"])
+    get_component_build(release_data["component_build"], model_build)
     try:
-        session.add(component_build)
+        #session.add(component_build)
```
Comment (CodeGat) [inline tools/release_provenance/save_release.py:60]: Can remove this comment

**Review by CodeGat — DISMISSED** [review]
Looks good to me

File: tools/release_provenance/save_release.py (hunk 1 above)
Comment (utkarshgupta95) [reply tools/release_provenance/save_release.py:60]: yea removing it

**Review by CodeGat — APPROVED** [review]

**Pull request merged**